README.md

```diff
@@ -11,12 +11,33 @@ Tekton !
 
 Download the latest binary executable for your operating system:
 
-// Note:- Need to update below links.
-* [Max OS X](README.md)
-* [Linux AMD 64](README.md)
-* [Windows AMD 64](README.md)
+* [Max OS X](https://github.com/tektoncd/cli/releases/download/v0.1.0/tektoncd-cli_0.1.0_Darwin_x86_64.tar.gz)
+
+  ```shell
+  # Get the tar.xz
+  curl -LO https://github.com/tektoncd/cli/releases/download/v0.1.0/tektoncd-cli_0.1.0_Darwin_x86_64.tar.gz
+  # Extract tkn in your PATH (e.g. /usr/local/bin)
+  sudo tar xvzf tektoncd-cli_0.1.0_Darwin_x86_64.tar.gz -C /usr/local/bin tkn
+  ```
+  
+* [Linux AMD 64](https://github.com/tektoncd/cli/releases/download/v0.1.0/tektoncd-cli_0.1.0_Linux_x86_64.tar.gz)
+  
+  ```shell
+  # Get the tar.xz
+  curl -LO https://github.com/tektoncd/cli/releases/download/v0.1.0/tektoncd-cli_0.1.0_Linux_x86_64.tar.gz
+  # Extract tkn in your PATH (e.g. /usr/local/bin)
```

Comment:
I think $HOME/bin/ would actually be a bit better but explaining how to add $PATH/bin to your path may be too much out of scope,